src/entities/identity/model/hooks/useRegistrationMutation.ts

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+
+import { useLoginMutation } from '@app/entities/identity';
+import { useSignUpMutation } from '@app/entities/identity';
+import { BaseError } from '@app/shared/api';
+
+import { IdentityModel } from '../..';
+
+type RegistartionData = {
+  email: string;
+  fullName: string;
+  password: string;
+};
+
+type UseRegistrationResult = {
+  isLoading: boolean;
+  error?: BaseError | null;
+  mutate: (data: RegistartionData) => void;
+};
+
+export const useRegistrationMutation = (
+  onLoginSuccess: () => void,
```

Comment:
I would rename it to just `onSuccess`. It implies that the whole registration process was successful

Reply:
ok